pyat/at/load/allfiles.py

```diff
@@ -38,10 +38,7 @@ def load_lattice(filepath, **kwargs):
     Known extensions are:
     """
     _, ext = os.path.splitext(filepath)
-    try:
-        return _load_extension[ext.lower()](filepath, **kwargs)
-    except KeyError:
```

Comment:
Do you think that the `KeyError` message is clear enough ? If so, you can apply the same simplification to `save_lattice`.